src/plugins/moreStickers/index.tsx

```diff
@@ -0,0 +1,204 @@
+                }
+            }, {
+                match: /(let \w=)(\w\.useCallback\(\(\)=>\{\(0,\w+\.\w+\)\([\w\.]*?\.STICKER,.*?);/,
+                replace: (_, decl, cb) => {
+                    const newCb = cb.replace(/(?<=\(\)=>\{\(.*?\)\().+?\.STICKER/, "\"stickers+\"");
+                    return `${decl}arguments[0]?.stickersType?${newCb}:${cb};`;
+                }
+            }, {
+                match: /(\w)=((\w)===\w+?\.\w+?\.STICKER)/,
+                replace: (_, isActive, isStickerTab, currentTab) => {
+                    const c = "arguments[0].stickersType";
+                    return `${isActive}=${c}?(${currentTab}===${c}):(${isStickerTab})`;
+                }
+            }]
+        },
+        {
+            find: '"ChannelTextAreaButtons"',
+            replacement: {
+                match: /,\(null===\(\w=\w\.stickers\)\|\|void 0.*?(\w)\.push\((\(0,\w\.jsx\))\((.+?),{disabled:\w,type:(\w)},"sticker"\)\)/,
+                replace: (m, _, jsx, compo, type) => {
+                    const c = "arguments[0].type";
+                    return `${m},${c}?.submit?.button&&${_}.push(${jsx}(${compo},{disabled:!${c}?.submit?.button,type:${type},stickersType:"stickers+"},"stickers+"))`;
+                }
+            }
+        },
+        {
+            find: ".Messages.EXPRESSION_PICKER_GIF",
+            replacement: {
+                match: /role:"tablist",.+?\.Messages\.EXPRESSION_PICKER_CATEGORIES_A11Y_LABEL,children:(\[.*?\)\]}\)}\):null,)(.*?closePopout:\w.*?:null)/s,
+                replace: m => {
+                    const stickerTabRegex = /(\w+)\?(\(.+?\))\((.{1,2}),.*?isActive:(\w)==.*?children:(.{1,10}Messages.EXPRESSION_PICKER_STICKER).*?:null/s;
+                    const res = m.replace(stickerTabRegex, (_m, canUseStickers, jsx, tabHeaderComp, currentTab, stickerText) => {
+                        const isActive = `${currentTab}==="stickers+"`;
+                        return (
+                            `${_m},${canUseStickers}?` +
+                            `${jsx}(${tabHeaderComp},{id:"stickers+-picker-tab","aria-controls":"more-stickers-picker-tab-panel","aria-selected":${isActive},isActive:${isActive},autoFocus:true,viewType:"stickers+",children:${jsx}("div",{children:${stickerText}+"+"})})` +
+                            ":null"
+                        );
+                    });
+
+                    return res.replace(/:null,((\w)===.*?\.STICKER&&\w+\?(\(.*?\)).*?(\{.*?,onSelectSticker:.*?\})\):null)/s, (_, _m, currentTab, jsx, props) => {
+                        return `:null,${currentTab}==="stickers+"?${jsx}($self.moreStickersComponent,${props}):null,${_m}`;
+                    });
+                }
+            }
+        },
+        {
+            find: '==="remove_text"',
+            replacement: {
+                match: /,\w\.insertText=\w=>{[\w ;]*?1===\w\.length&&.+?==="remove_text"/,
```

Comment:
```suggestion
                match: /,\i\.insertText=\i=>{1===\i\.length&&.+?==="remove_text"/,
```
is `[\w ;]*?` needed? because i dont see anything there

Reply:
I'm not sure if it is required in the current version.
In the previous version, they made a space/change line here (their minifier did this? dunno)